dev-env

```diff
@@ -72,31 +72,41 @@ mkdir -p "$GAIA_DATA" && cd "$GAIA_DATA" && cd ../
 
 ONE_CHAIN="$(dirname "$0")/one-chain"
 
-"$ONE_CHAIN" gaiad "$chainid0" ./data 26657 26656 6060 9090
-"$ONE_CHAIN" gaiad "$chainid1" ./data 26557 26556 6061 9091
+CHAIN_0_RPC_PORT=26657
+CHAIN_0_RPC_ADDR="localhost:$CHAIN_0_RPC_PORT"
+CHAIN_1_RPC_PORT=26557
+CHAIN_1_RPC_ADDR="localhost:$CHAIN_1_RPC_PORT"
 
-[ -f "$GAIA_DATA/$chainid0.log" ] && echo "$chainid0 initialized. Watch file $GAIA_DATA/$chainid0.log to see its execution."
-[ -f "$GAIA_DATA/$chainid1.log" ] && echo "$chainid1 initialized. Watch file $GAIA_DATA/$chainid1.log to see its execution."
+CHAIN_0_SAMOLEANS=100
+CHAIN_1_SAMOLEANS=100
```

Comment:
Can you increase this to something like `100000000000`. 
`100` is not enough for packet testing requirements.